charts/ceph-csi-cephfs/values.yaml

```diff
@@ -28,6 +28,7 @@ serviceAccounts:
 #     cephFS:
 #       subvolumeGroup: "csi"
 #       netNamespaceFilePath: "{{ .kubeletDir }}/plugins/{{ .driverName }}/net"
+#       radosNamespace: "app-1"
```

Comment:
`app-1` to `csi` lets use some meaningful names

Reply:
done